docs/src/modules/components/overview/mainDemo/DigitalClock.tsx

```diff
@@ -58,7 +57,7 @@ export default function DigitalClock() {
               hours: renderMultiSectionDigitalClockTimeView,
               minutes: renderMultiSectionDigitalClockTimeView,
               meridiem: renderMultiSectionDigitalClockTimeView,
-            } as TimePickerViewRenderers<Dayjs, 'meridiem' | TimeView, any>
+            } as any
```

Comment:
We are only using such a hacky approach on the docs demo, where code is not shown.
I would prefer avoiding public exporting of the used type until there is a clear need for it.
WDYT @noraleonte, does that make sense to you? 🤔

Reply:
Couldn't you do the following?
 
```suggestion
            } as StaticTimePickerProps<Dayjs>['viewRenderers']
```

Reply:
You are right, we could, but do you think that it matters what we cast to in an example, which is not visible to the end user? 🤔

Reply:
we ourselves could benefit from the better type safety.

Reply:
Sure, it's clearly better than `as any` and will at least have some safeguards. :)

Reply:
Fixed. 👌